proto/ojo/oracle/v1/tx.proto

```diff
@@ -107,6 +111,38 @@ message MsgGovUpdateParams {
 // MsgGovUpdateParamsResponse defines the Msg/GovUpdateParams response type.
 message MsgGovUpdateParamsResponse {}
 
+
+// MsgGovAddDenoms defines the Msg/GovAddDenom request type.
+message MsgGovAddDenoms {
+  option (gogoproto.equal)            = true;
+  option (gogoproto.goproto_getters)  = false;
+  option (gogoproto.goproto_stringer) = false;
+  option (cosmos.msg.v1.signer)       = "authority";
+
+  // authority is the address of the governance account.
+  string authority = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+
+  // title of the proposal
+  string title = 2;
+
+  // description of the proposal
+  string description = 3;
+
+  // height at which the param update must be performed
+  int64 height   = 4;
+
+  repeated Denom denom_list = 5 [
+    (gogoproto.castrepeated) = "DenomList",
+    (gogoproto.nullable) = false
+  ];
+
+  // whether or not the asset should be mandatory
+  bool mandatory = 6;
```

Comment:
Having this be a bool makes it so we can't add some denoms to be in both the accept list and mandatory list and some in only the accept list in the same message. However, I'm okay with it for the purpose of simplifying the msg if you don't think we'll be doing the second case too often (multiple msg prop probably won't be the end of the world anyways).

Reply:
Iirc the parameter logic is that the mandatory list is always a subset of the acceptlist; if someone passes in Mandatory = true for a new asset, it'll be added to both. Please double check me here